packages/grid/src/__tests__/__e2e__/Grid.cy.tsx

```diff
@@ -606,4 +556,44 @@ describe("Grid", () => {
     checkRowSelected(2, true);
     checkRowSelected(3, true);
   });
+
+  describe("Column Sorting", () => {
+    it.only("should sort column values when sorting is enabled", () => {
+      cy.mount(<RowSelectionModes />);
+
+      // first click: sort in ascending order
+      cy.findAllByTestId("column-header")
+        .eq(2)
+        .should("have.text", "B")
+        .as("sortableHeader")
```

Comment:
What role does this `as()` have? Usually, it would "save" a query for later, but here we don't use it later.

Reply:
removed that - tbh I forgot why it was added initially but it makes sense, reading other use cases of `as()` it's not needed in here